.github/workflows/checkstyle.yml

```
@@ -1,30 +1,43 @@
-name: Checkstyle
-
-on:
-  pull_request:
-
-permissions:
-  contents: read
-
-jobs:
-  checkstyle:
-
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v3
-    - name: Set up JDK 17
-      uses: actions/setup-java@v3
-      with:
-        java-version: '17'
-        distribution: 'temurin'
-    - name: Run check style
-      uses: nikitasavinov/checkstyle-action@master
-      if: always()
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        reporter: 'github-pr-check'
-        tool_name: 'reviewdog'
-        checkstyle_config: 'config/checkstyle/checkstyle.xml'
-        workdir: 'src/main'
-        level: error
+name: Checkstyle
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  checkstyle:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+    - name: Run check style
+      uses: nikitasavinov/checkstyle-action@master
+      if: always()
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        reporter: 'github-pr-check'
+        tool_name: 'reviewdog'
+        checkstyle_config: 'config/checkstyle/checkstyle.xml'
+        workdir: 'src/main'
+        level: error
+
+    - name: Discord Notification
+      uses: sarisia/actions-status-discord@v1
+      if: always()
+      with:
+        webhook: ${{ secrets.DISCORD_WEBHOOK }}
+        status: ${{ job.status }}
+        content: "Hey <@everyone>"
+        title: "deploy"
+        description: "Build and deploy to GitHub Pages"

```

実施内容の実態に合わせたメッセージがよいですね。
```suggestion
        title: "Checkstyle"
        description: "Rucn Checkstyle"

```